src/main/java/com/neovisionaries/ws/client/OkHostnameVerifier.java

```diff
@@ -58,7 +58,6 @@
   private OkHostnameVerifier() {
   }
 
-  @Override
```

Comment:
To ensure compatibility with Java 1.5 (@Override is not allowed for implementations of interface methods).